SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -180,6 +183,16 @@ Worker_RequestId USpatialSender::CreateEntity(USpatialActorChannel* Channel)
 	ComponentDatas.Add(SpawnData(Actor).CreateSpawnDataData());
 	ComponentDatas.Add(UnrealMetadata(StablyNamedObjectRef, ClientWorkerAttribute, Class->GetPathName(), bNetStartup).CreateUnrealMetadataData());
 
+	if(TArray<TSharedRef<FPendingRPCParams>>* RPCList = OutgoingOnCreateEntityRPCs.Find(Actor))
+	{
+		RPCsOnEntityCreation QueuedRPCs = PackQueuedRPCsForActor(RPCList, Actor);
```

Comment:
Looking at this a bit more, I think you should dereference the RPCList and make the PackQueuedRPCsForActor take a const ref to the array.